doc/user/ospfd.rst

```diff
@@ -449,6 +449,43 @@ Areas
     Prevents an *ospfd* ABR from injecting inter-area
     summaries into the specified stub area.
 
+.. index:: area A.B.C.D nssa
+.. clicmd:: area A.B.C.D nssa
+
+.. index:: area (0-4294967295) nssa
+.. clicmd:: area (0-4294967295) nssa
+
+.. index:: no area A.B.C.D nssa
+.. clicmd:: no area A.B.C.D nssa
+
+.. index:: no area (0-4294967295) nssa
+.. clicmd:: no area (0-4294967295) nssa
```

Comment:
Could be simplify by adding `[no]` in front of the command like `.. index:: [no] area A.B.C.D nssa`

Reply:
oh, no, please don't ask folks to do that: the result is that the index entries are placed together at "no", instead of the actual first keyword.

Reply:
OK. But in this case, we need to be coherent as I saw some PR and some doc that used the `[no]` wording.

Can we have a clear status about how to write CLI documentation?